### ISSUE TITLE
This app fails to start on Windows startup

### ISSUE THREAD

**Issue by L1cardo** [karpach/remote-shutdown-pc#26]
I can see there is remote-shutdown-pc on the startup page, but the app won't start actually and there is no icon on system tray, so non of the commands work.

I am on Windows 11 22H2.

The app is v1.1.2

![image](https://user-images.githubusercontent.com/33802186/197824905-c37a1cfb-b584-4b54-88b8-732e11f5e08a.png)

**Comment by karpach** [comment]
Does it start if you click on the executable? I didn't test it on Windows 11, I am not sure if it works.

**Comment by L1cardo** [comment]
> Does it start if you click on the executable? I didn't test it on Windows 11, I am not sure if it works.


Yes, it works fine if I start it manually.

**Comment by L1cardo** [comment]
hello，how to solve this?

**Comment by karpach** [comment]
I finally got a copy of Windows 11 Pro Version 22H2. I unchecked / checked "Auto load at Windows startup" checkbox and the program successfully started after Windows restart.

Try to run the application as Administrator. Uncheck / check setting. The option updates the Windows registry: SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Run, which is a well-known way of making things run at startup. Maybe you have some kind of Antivirus, that prevents Windows registry updates.
